accounts/views.py

```diff
@@ -0,0 +1,46 @@
+from django.contrib.auth import login, logout
+from django.contrib.auth.forms import AuthenticationForm
+from django.core.urlresolvers import reverse, reverse_lazy
+from django.http import HttpResponseRedirect
+from django.views import generic
+from django.views.generic import TemplateView
+
+from . import forms
+
+
+class LoginView(generic.FormView):
+    form_class = AuthenticationForm
+    success_url = reverse_lazy('accounts:home')
+    template_name = 'accounts/login.html'
+
+    def get_form(self, form_class=None):
+        if form_class is None:
+            form_class = self.get_form_class()
+        return form_class(self.request, **self.get_form_kwargs())
+
+    def form_valid(self, form):
+        login(self.request, form.get_user())
+        return super().form_valid(form)
+
+
+class LogoutView(generic.RedirectView):
+    url = reverse_lazy('home')
+
+    def get(self, request, *args, **kwargs):
```

Comment:
I would put this on `dispatch` instead of `get`. Generally `dispatch` is the best method to use if you just want to ensure that something is run whenever the endpoint is accessed, since it's run regardless of the HTTP verb.